osu.Game/Screens/OnlinePlay/Match/RoomSubScreen.cs

```diff
@@ -473,8 +476,20 @@ private void cancelTrackLooping()
         /// <param name="room">The room to change the settings of.</param>
         protected abstract RoomSettingsOverlay CreateRoomSettingsOverlay(Room room);
 
-        public class UserModSelectButton : PurpleTriangleButton
+        public class UserModSelectButton : PurpleTriangleButton, IKeyBindingHandler<GlobalAction>
         {
+            public virtual bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
```

Comment:
There's no actual reason why any of the two methods added in this pull should be virtual.